posthog/tasks/usage_report.py

```diff
@@ -817,6 +838,12 @@ def _get_team_report(all_data: Dict[str, Any], team: Team) -> UsageReportCounter
         event_explorer_api_duration_ms=all_data["teams_with_event_explorer_api_duration_ms"].get(team.id, 0),
         survey_responses_count_in_period=all_data["teams_with_survey_responses_count_in_period"].get(team.id, 0),
         survey_responses_count_in_month=all_data["teams_with_survey_responses_count_in_month"].get(team.id, 0),
+        billable_data_warehouse_rows_synced_in_period=all_data["teams_with_data_warehouse_rows_synced_in_period"].get(
```

Comment:
@EDsCODE why is this `billable_`? did you just copy from feature flags? feature flags is a bit strange since local eval is billed at 10x so the final value is the `billable_` one but for this it's not necessary.

Reply:
Yeah copies to stay consistent with latest format but will remove! Thanks for the clarification